gateway/Dockerfile

```diff
@@ -1,20 +1,21 @@
-FROM python:3.10.12-slim-buster
+FROM registry.access.redhat.com/ubi9-minimal:9.4-1134
+RUN microdnf install -y python3.11-3.11.7-1.el9_4.1 python3.11-pip-22.3.1-5.el9 python3.11-devel-3.11.7-1.el9_4.1
+RUN ln -s /usr/bin/python3.11 /usr/local/bin/python3 && \
+    ln -s /usr/bin/python3.11 /usr/local/bin/python &&\
+    ln -s /usr/bin/pip3.11 /usr/local/bin/pip3 &&\
+    ln -s /usr/bin/pip3.11 /usr/local/bin/pip
 
 WORKDIR /usr/src/app
 
 # set environment variables
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-# install psycopg2 dependencies
-RUN apt-get -y update &&\
-    apt-get install --no-install-recommends -y gcc=4:8.3.0-1 python3-dev=3.7.3-1 vim=2:8.1.0875-5+deb10u6 &&\
```

Comment:
Is it possible to keep vi/vim in this image? (we've used it for live debugging in the past)
or is there an editor in the UBI already?